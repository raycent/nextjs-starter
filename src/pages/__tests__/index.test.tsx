import { render } from 'src/tests/testUtils';
import IndexPage from 'src/pages/index';

describe('IndexPage', () => {
  it('matches snapshot', () => {
    const { asFragment } = render(<IndexPage />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
