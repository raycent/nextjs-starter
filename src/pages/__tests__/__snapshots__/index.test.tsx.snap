// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IndexPage matches snapshot 1`] = `
<DocumentFragment>
  0
  <div>
    <button>
      -
    </button>
    <button>
      +
    </button>
    <button>
      Reset
    </button>
  </div>
</DocumentFragment>
`;
